app/src/main/res/values-gl/strings.xml

```diff
@@ -2,97 +2,96 @@
-    <string name="monospaced_font">Use monospaced font</string>
+    <string name="monospaced_font">Usar a fonte de monoespaciado</string>
     <string name="show_keyboard">Mostrar teclado ao inicio</string>
     <string name="show_word_count">Mostrar conta de palabras</string>
-    <string name="show_character_count">Show character count</string>
-    <string name="alignment">Aliñamento</string>
+    <string name="show_character_count">Amosar a contaxe dos caracteres</string>
+    <string name="alignment">Alineamento</string>
     <string name="left">Esquerda</string>
     <string name="center">Centro</string>
     <string name="right">Dereita</string>
     <string name="widget_note">Nota a utilizar no widget</string>
     <string name="show_note_picker">Mostrar selector de nota ao inicio</string>
     <string name="autosave_notes">Gardado automático</string>
     <string name="enable_line_wrap">Habilitar sangrado de liña</string>
-    <string name="use_incognito_mode">Use Incognito mode of keyboards</string>
-    <string name="move_undone_checklist_items">Move undone checklist items at the top</string>
+    <string name="use_incognito_mode">Usar o modo incógnito dos teclados</string>
+    <string name="move_undone_checklist_items"> Mover os elementos dunha lista de verificación non rematada a arriba</string>
 
     <!-- Checklists -->
-    <string name="checklist">Checklist</string>
-    <string name="add_new_checklist_item">Add a new checklist item</string>
-    <string name="add_new_checklist_items">Add new checklist items</string>
-    <string name="checklist_is_empty">The checklist is empty</string>
+    <string name="checklist">Lista de verificación</string>
+    <string name="add_new_checklist_item">Engadir un elemento a unha nova lista de verificación</string>
+    <string name="add_new_checklist_items">Engadir elementos a unha nova lista de verificación</string>
+    <string name="checklist_is_empty">A lista de verificación está baleira</string>
 
     <!-- Import / Export -->
     <string name="export_all_notes">Exportar todas as notas como ficheiros</string>
     <string name="import_notes">Importar varios ficheiros como notas</string>
-    <string name="import_folder">Import folder</string>
+    <string name="import_folder">Importar cartafoles</string>
 
     <!-- FAQ -->
-    <string name="faq_1_title">How can I change the widgets color?</string>
-    <string name="faq_1_text">In case you have only 1 active widget, you can either recreate it, or use the button in the app settings for customizing it. If you have multiple active widgets, the button in the app settings will not be available. As the app supports color customization per-widget, you will have to recreate the widget that you want to customize.</string>
-
+    <string name="faq_1_title">Como cambio a cor dos widgets?</string>
+    <string name="faq_1_text">No caso de que teñas só un widget activo, podes crealo de novo ou usar o botón nos axustes do app para personalizalo.
```

Comment:
@davidre345 @tibbi `</string>` is missing.
Should be:
```
<string name="faq_1_text">No caso de que teñas só un widget activo, podes crealo de novo ou usar o botón nos axustes do app para personalizalo.</string>
```

Reply:
create a pull request please with all updates